metrics/SystemMetrics.md

```diff
@@ -0,0 +1,38 @@
+# System Metrics
+This document serves to document the "look and feel" of the OpenCensus API for auto-collection of system metrics such as processes, memory & CPU usage, disks, etc.
+
+## Supported platforms
+TODO: Add list of supported platforms.
+
+## Common metrics collection
+These are some default metrics supported by OpenCensus system metrics package/contrib. This is to ensure consistency across language implementations, as far as practical.
+
+| Metric Name                               | Unit  | Type                       | Description                                                        | Labels                  |
+|-------------------------------------------|-------|----------------------------|--------------------------------------------------------------------|-------------------------|
+| system/cpu_seconds/total                  | s     | Int64 Cumulative           | Total kernel/system user CPU in seconds.                           | Hostname, Mode          |
```

Comment:
@rghetia can you add your idea about the idle/total/user label idea?

Reply:
It is already included here but I would rephrase it differently.

system/cpu_seconds  | s  | Int64 Cumulative |  System CPU in seconds | Hostname, Mode = { total, idle, system, nice, user, iowait, stolen}

Similarly other metrics can be redefined with common metric and possible Label and its values.

Reply:
Sure, done.